### DISCUSSION TITLE
waterfox G6 update

### DISCUSSION THREAD

**Discussion by fanhir** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#73] [Q&A] [answered]
Can we please have Waterfox G6 in the debian repositories ?

**Comment by hawkeye116477** [comment]
It will be available again when I'll resolve dependency issues.

**Comment by hawkeye116477** [comment, marked as answer]
I've good news, problems fixed. Packages for Debian 10, 11 and 12 are now available.

**Comment by fanhir** [comment]
Thanks. Is there a reason you didn't also include it in the Debian_Testing repository?

**Reply by hawkeye116477** [reply]
Packages for Debian Testing and Unstable will be available after weekend.
